src/sparkx/Filter.py

```diff
@@ -820,7 +820,7 @@ def multiplicity_cut(
     particle_list:
         List with lists containing particle objects for the events
 
-   cut_value_tuple : tuple
+    cut_value_tuple : tuple
```

Comment:
Was this on purpose?

Reply:
Yes, this was on purpose. The line was not aligned with the `particle_list` above.